child/_child_galaxy_video.Rmd

```diff
@@ -0,0 +1,21 @@
+
+Here is a video tutorial that describes the basics of using Galaxy on AnVIL.
+
+<iframe src="https://drive.google.com/file/d/16QEY8x-gBsUkKEeO3w_H-I4SLIBPloXd/preview" width="640" height="360" allow="autoplay"></iframe>
+
+### Objectives
+
+- Start compute for your Galaxy on AnVIL
+- Run tool to quality control sequencing reads
+- Stop compute to minimize expenses
+
+### Slides
+
+The slides for this tutorial are are located [here](https://docs.google.com/presentation/d/1yYCg4cPVBMMDghT17B4XzROieqyMH99Ex9nMm_Scm9Q).
+
+## Galaxy: Step-by-step guide
```

Comment:
I think delete this section, in case somebody doesn't want to do the step-by-step guide right after the video.

I left a suggestion to move the last comment (that you need access to a Workspace) to the beginning of the guide on starting galaxy